src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/files/jupyterlab/overrides.json

```diff
@@ -9,6 +9,7 @@
         "authToken": ""
     },
     "@jupyterlab/apputils-extension:notification": {
-        "fetchNews": "false"
+        "fetchNews": "false",
+        "checkForUpdates": "false"
```

Comment:
@viniciusdc shouldn't this be a boolean value (i.e., `false`) rather than a string? Looking at the default values, it is indeed a boolean. I mean, https://github.com/jupyterlab/jupyterlab/blob/13ba4cc6dad700b5df13beb8c39fb0b90bb6b1ea/packages/apputils-extension/src/announcements.ts#L222 should convert it to boolean anyways, but somehow it just feels wrong.

Makes me wonder whether `"fetchNews"` should also have a boolean value.

Reply:
Apparently, `"fetchNews"` does not let you pass a boolean value. However, `"checkForUpdates"` complains if you pass a string:

![image](https://github.com/nebari-dev/nebari/assets/36754005/5690a9ed-f2a2-476a-853f-5e157434626d)

So much for consistency!

Reply:
@viniciusdc I'm changing this and will merge

Reply:
thanks, @marcelovilla. indeed, I went with string because of ` fetchNews.` Also, good catch. I remember spotting the error message above and manually updating the pod, but I didn't commit the change to Bolean. Thanks